src/components/ProjectWork/ContactSection.js

```diff
@@ -1,46 +1,88 @@
+  const [copyButtonText, setCopyButtonText] = useState(defaultCopyButtonText);
 
   const copyEmail = () => {
+    // Copy text
     var textArea = document.createElement('textarea');
     textArea.value = emailContent;
     document.body.appendChild(textArea);
     textArea.select();
     document.execCommand('copy');
     document.body.removeChild(textArea);
+    //Change button text
+    setCopyButtonText('Copied!');
+    //Change button text to default text after delay
+    setTimeout(() => {
+      setCopyButtonText(defaultCopyButtonText);
+    }, 5000); //5 seconds
+  };
+
+  const openMailClient = () => {
+    const subjectLine = `Interest In Project: ${projectTitle}`;
+    const body = Template(project, 'uri');
+    window.location.href = `mailto:${orgEmail}?subject=${subjectLine}&body=${body}`;
   };
 
+  const ContactEmailTemplate = () => (
+    <div>
+      <div className="info-container">
+        <h4>Email Template</h4>
+        <p>
+          This is a sample email template that you may use to contact the client. If you wish to use it, please modify
+          the information within the brackets in order to tailor it to the project and client. If you are in a team, be
+          sure to include them in the email as well.
+        </p>
+      </div>
+      <Card className="project-detail-email">
+        <div className="buttons-container">
+          <Button
+            className={`primary-button btn ${copyButtonText !== defaultCopyButtonText ? 'clicked' : ''}`}
+            onClick={copyEmail}
+          >
+            <i className="far fa-copy" aria-hidden="true"></i>
+            {copyButtonText}
+          </Button>
+          <Button className="primary-button btn" onClick={openMailClient}>
+            <i className="fas fa-envelope-open-text" aria-hidden="true"></i>
+            Open in Mail Client
+          </Button>
+        </div>
+        <div>{emailContent}</div>
+      </Card>
+    </div>
+  );
+
   return (
     <div>
-      <h4>Contact information</h4>
-      {hasConfirmed ? (
-        <>
-          {orgEmail && <p>Email: {orgEmail}</p>}
-          {orgPhone && <p> Phone number: {orgPhone}</p>}
-          <h4>Email template</h4>
-          <div className="confirmation-btn-container">
-            <Button className="secondary-button btn" onClick={copyEmail}>
-              Copy template
-            </Button>
-          </div>
-          <p className="project-detail-email">{emailContent}</p>
-        </>
-      ) : (
-        <p>Please agree to the anti-racism commitment in step 1 to view contact information</p>
-      )}
+      <div className={hasConfirmed ? 'title-container' : ''}>
+        <h3>Contact Information</h3>
+        {hasConfirmed ? (
+          <>{orgEmail && <p>Email: {orgEmail}</p>}</>
+        ) : (
+          <p className="mb-0">Please agree to the anti-racism commitment in step 1 to view contact information</p>
+        )}
+      </div>
+      {hasConfirmed && <ContactEmailTemplate />}
```

Comment:
`<ContactEmailTemplate />` can just go up with line 80 since it's the same condition